src/Stratis.Bitcoin/Builder/FullNodeBuilder.cs

```diff
@@ -197,9 +178,21 @@ public IFullNode Build()
             if (fullNode == null)
                 throw new InvalidOperationException("Fullnode not registered with provider");
 
-            fullNode.Initialize(new FullNodeServiceProvider(
-                fullNodeServiceProvider,
-                this.Features.FeatureRegistrations.Select(s => s.FeatureType).ToList()));
+            List<Type> features = this.Features.FeatureRegistrations.Select(s => s.FeatureType).ToList();
+
+            // Create configuration file if required
+            this.NodeSettings?.CreateDefaultConfigurationFile(features, fullNodeServiceProvider);
```

Comment:
Dont we do that in the NodeSettings?

Reply:
`network` existing here just pass this in o to the method.

Reply:
Changed.